pkg/apis/vck/v1/types.go

```diff
@@ -95,8 +95,6 @@ type VolumeConfig struct {
 	ID           string              `json:"id"`
 	Replicas     int                 `json:"replicas"`
```

Comment:
Do we want the replicas to be here or in `options`?

Reply:
I think it should be here as it would be common to all the source types.